docs/model-dev-guide/hyperparameter/search-methods/hp-custom.rst

```diff
@@ -39,8 +39,7 @@ Both search runners execute the custom hyperparameter tuning algorithm and start
 experiment on a Determined cluster.
 
 The following sections explain the steps to take in order to implement and use a custom
-hyperparameter search algorithm. A detailed example can be found in
-:download:`asha_search_method.tgz </examples/asha_search_method.tgz>`.
+hyperparameter search algorithm.
```

Comment:
The following sections describe the steps needed to implement and use a custom
hyperparameter search algorithm.